### PR TITLE
[Cloudflare] New dashboards using Lens, and bumping format version

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

This PR remakes the existing dashboards into using Lens, and updates its styling and layout to match other new dashboards.
I also updated the format_version, and added pipeline errors.
Updated some extra fields from the test data to have better coverage.
Fixed some small bugs and added some small additions to the pipeline to fit the dashboards.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #https://github.com/elastic/security-team/issues/5910

## Screenshots

Overview:
cloudflare-overview1
Overview2

Clients:
Clients1

Content:
Content1

Reliability
Rel1
Rel2

Performance:
cloudflare-perf1
Perf2

Security overview:
cloudflare-sec1
Seco2

Ratelimit:
ratelimit1

WAF:
waf1
waf2

### CODE REVIEW

**Comment by efd6** [comment]
Minor suggestions/queries.

1. In the overview because of the alignment of the counters with the pie charts, it looks like they are associated pairwise (counter with pie chart). This is not true, so is there some way to visually break that association.
2. In ratelimit, the top left pie chart can be narrower, giving up space to the table.

Out-of-scope-concern: I'm still troubled by how lens counters reduce data impact by avoiding the central space.

**Comment by P1llus** [comment]
> Minor suggestions/queries.
> 
> 1. In the overview because of the alignment of the counters with the pie charts, it looks like they are associated pairwise (counter with pie chart). This is not true, so is there some way to visually break that association.
> 2. In ratelimit, the top left pie chart can be narrower, giving up space to the table.
> 
> Out-of-scope-concern: I'm still troubled by how lens counters reduce data impact by avoiding the central space.

Added in your changes, though I did not want to re-add my custom input config again just for some test data, I can show that the sizes and placements has changed:

Overview, grouped them side by-side instead, which still makes sense.
Overview changed

Rate Limit just made it less wide, but a little bit higher to compensate:
ratechanged

**Review by ShourieG — APPROVED** [review]
LGTM

File: packages/cloudflare/kibana/dashboard/cloudflare-5a5d6b80-49b9-11e9-bd1f-75f359ac0c3f.json [hunk 1]
```diff
@@ -19,1077 +17,991 @@
...
-                    "x": 1,
-                    "y": 33
+                    "w": 40,
+                    "x": 8,
+                    "y": 22
                 },
                 "panelIndex": "2",
-                "type": "visualization",
-                "version": "8.0.0"
+                "title": "[Cloudflare] Client Requests Methods Over Time",
```
Comment (kcreddy) [inline packages/cloudflare/kibana/dashboard/cloudflare-5a5d6b80-49b9-11e9-bd1f-75f359ac0c3f.json:430]:
```suggestion
                "title": "[Cloudflare] Client Requests by Methods Over Time",
```

**Review by kcreddy — APPROVED** [review]
Just some nitpicks. LGTM 📈

**Review by kcreddy — APPROVED** [review]
Just some nitpicks. LGTM 📈

**Pull request merged**